app/com/yetu/oauth2provider/views/registration.scala.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+			<title>Registration</title>
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/setup.css")">
```

Comment:
we should use the versioned Assets. (`Assets.versioned(` instead of `Assets.at(`) to load css and other things. 

You need the digest plugin, i.e. 
```
addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.0")
addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.0")
addSbtPlugin("com.typesafe.sbt" % "sbt-web" % "1.1.1")
``` 
in plugins.sbt, see https://github.com/yetu/controlcenter/blob/master/project/plugins.sbt
Then add them to the pipeline  stages (see build.sbt on  https://github.com/yetu/controlcenter/blob/master/build.sbt)

And in the routes file, change the line with the assets like in the https://github.com/yetu/controlcenter/blob/master/conf/routes 

More information:
https://playframework.com/documentation/2.3.x/Assets under Reverse routing and fingerprinting for public assets